recipes/recipes_emscripten/zenoh-pico/recipe.yaml

```diff
@@ -0,0 +1,63 @@
+context:
+  name: zenoh-pico
+  version: '1.4.0'
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/eclipse-zenoh/${{ name }}/archive/refs/tags/${{ version }}.tar.gz
+  sha256: 9cbe440cc1de8b70f48a61a5344120c0a83810e15da1fce45573b1041a8edc59
+
+build:
+  number: 0
+  skip: win
```

Comment:
There's no windows build. We can remove this line

Reply:
Done